petri/src/vm/openvmm/modify.rs

```diff
@@ -136,6 +136,19 @@ impl PetriVmConfigOpenVmm {
         self
     }
 
+    /// Enable TPM state persistence
+    pub fn with_tpm_state_persistence(mut self) -> Self {
+        let Some(ged) = &mut self.ged else {
```

Comment:
i'd probably just write this as expects? weird else block here if we're just going to panic. I dunno this just looks weird to me...

Reply:
updated